packages/env/src/index.ts

```diff
@@ -1,20 +1,21 @@
-import path from 'path'
-import { fileURLToPath } from 'url'
-
-import dotenv from 'dotenv'
-
-const DIRNAME = path.dirname(fileURLToPath(import.meta.url))
+import dotenv, { type DotenvPopulateInput } from 'dotenv'
 
 const env = process.env.NODE_ENV || 'development'
+const useLocalhost = process.env.USE_LOCALHOST === 'true'
 
 // Don't write production .env files!
 if (env !== 'production') {
-  const url = path.join(DIRNAME, `./env/${env}.env`)
-  const result = dotenv.config({
-    path: url,
-  })
+  const dbHost = useLocalhost ? `localhost` : `db`
+  const redisHost = useLocalhost ? `localhost` : `redis`
 
-  if (result.error) {
-    throw result.error
-  }
+  dotenv.populate(process.env as DotenvPopulateInput, {
+    NODE_ENV: env,
+    DATABASE_URL: `postgres://latitude:secret@${dbHost}:5432/latitude_development`,
+    ELASTIC_URL: 'https://elastic:9200',
+    ELASTIC_USERNAME: 'latitude',
+    ELASTIC_PASSWORD: 'secret',
+    REDIS_PORT: '6379',
+    REDIS_HOST: redisHost,
+    REDIS_PASSWORD: 'secret',
+  })
```

Comment:
Did a bit of DRY and added a way to configure the correct host depending on whether the package is being ran locally or on a docker image